third_party/terraform/tests/resource_access_context_manager_service_perimeter_test.go.erb

```diff
@@ -63,6 +63,14 @@ func testAccAccessContextManagerServicePerimeter_updateTest(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				Config: testAccAccessContextManagerServicePerimeter_updateDryrun(org, "my policy", "level", "perimeter"),
```

Comment:
Can you add a case unsetting the dry run? It's possible the boolean will need `send_empty_value: true` set on it so that we send a `false` value in the PATCH request (this API uses an update mask, so it may not though!)

I'm also curious what the behaviour of `spec` is once the field in unset, since it doesn't appear in requests until it's set.

Reply:
Added a new step to the test reverting the config to before the dry-run (unsetting it). Works fine due to the update mask.